api/api.js

```diff
@@ -517,6 +522,16 @@ async function getClusterInfo() {
     0,
   );
 
+  votingAll = _.map(votingAll, e => {
+    const [, v] = e;
+
+    let voteAccount = solanaWeb3.VoteAccount.fromAccountData(v.data);
+    voteAccount.nodePubkey = voteAccount.nodePubkey.toString();
```

Comment:
Note that there's nothing stopping somebody from creating multiple vote accounts with the same `nodePubkey` (this likely would happen if somebody is having a hard time booting a validator and clears their config a couple times), so we probably want to prune duplicates from `votingAll`.  We don't have a good way to pick one dup over another right now, so just pick one :)

Reply:
@mvines got it. I will come back to this momentarily (have created a tracking issue for it on the board)